packages/connect-popup/package.json

```diff
@@ -11,6 +11,7 @@
     },
     "dependencies": {
         "@trezor/connect": "9.0.0-beta.5",
+        "@trezor/components": "*",
```

Comment:
You can try to add `sideEffects: false` to components package.json, it should enable treeshaking so it should not bundle whole components package into your final bundle

Reply:
ok, lets see what happens 29f89c98e4d666089677f07cc5a6f3ae878563f5